compiler/plugins/target/WebGPU/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Tint is needed to compile SPIR-V into WGSL source code.
+# Tint also requires SPIRV-Tools, which requires SPIRV-Headers.
+#
+# NOTE: these can be synced by referencing one of these repositories:
+#   * https://dawn.googlesource.com/dawn/
+#   * https://dawn.googlesource.com/tint/
+#   * https://chromium.googlesource.com/vulkan-deps/+/refs/heads/main/DEPS
+# or they can be updated independently
+set(IREE_TINT_TAG          "fdb8787e9c1b79770bd98a8faf37fbe48a3077a4")  # 2023-03-06
+set(IREE_SPIRV_HEADERS_TAG "b730938c033ede3572b660ab019b438509ba24d9")  # 2023-08-10
+set(IREE_SPIRV_TOOLS_TAG   "43b8886490eb6af81fc61e0ff071c51a922af864")  # 2023-08-11
+message(STATUS "Configuring WebGPU target deps (SPIRV-Headers, SPIRV-Tools, Tint)")
+list(APPEND CMAKE_MESSAGE_INDENT "  ")
+add_subdirectory(spirv-headers EXCLUDE_FROM_ALL)
+add_subdirectory(spirv-tools EXCLUDE_FROM_ALL)
+add_subdirectory(tint EXCLUDE_FROM_ALL)
+list(POP_BACK CMAKE_MESSAGE_INDENT)
```

Comment:
Sample logs: https://github.com/openxla/iree/actions/runs/6895343970/job/18759132318?pr=15612#step:4:514

```
-- Registering static linked compiler plugin 'hal_target_cuda' (iree::compiler::plugins::target::CUDA::CUDA)
-- Configuring WebGPU target deps (SPIRV-Headers, SPIRV-Tools, Tint)
--   Tint build samples: OFF
--   Tint build docs: OFF
--   Tint build docs with warn as error: OFF
--   Tint build SPIR-V reader: ON
--   Tint build WGSL reader: OFF
--   Tint build GLSL writer: OFF
--   Tint build HLSL writer: OFF
--   Tint build MSL writer: OFF
--   Tint build SPIR-V writer: OFF
--   Tint build WGSL writer: ON
--   Tint build IR: ON
--   Tint build fuzzers: OFF
--   Tint build SPIRV-Tools fuzzer: OFF
--   Tint build AST fuzzer: OFF
--   Tint build regex fuzzer: OFF
--   Tint build benchmarks: OFF
--   Tint build tests: OFF
--   Tint build with ASAN: OFF
--   Tint build with MSAN: OFF
--   Tint build with UBSAN: OFF
--   Tint build checking [chromium-style]: OFF
--   Tint build remote-compile tool: OFF
--   Using python3
--   Found PythonInterp: /usr/bin/python3 (found suitable version "3.9.5", minimum required is "3") 
-- Registering static linked compiler plugin 'hal_target_webgpu' (iree::compiler::plugins::target::WebGPU::WebGPU)
```

(Similar indentation changes are in https://github.com/openxla/iree/pull/15596)

Reply:
Neat! I've learnt a trick. :)